infrastructure/testdriver/click_iframe.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>TestDriver click method</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<iframe src="click_child.html"></iframe>
+
+<script>
+async_test(t => {
+  addEventListener("load", t.step_func(() => { 
```

Comment:
I notice all of the tests have this outer load listener - is it necessary? (I.e. what are we waiting for?)

Reply:
For the iframes to load